devfiles/vscode-extensions.devfile.yaml

```diff
@@ -0,0 +1,79 @@
+# A devfile to develop VS Code extensions.
+# For developers who would like to create his plugin inside Che-Theia.
+
+apiVersion: 1.0.0
+metadata:
+  name: che-theia-all
+
+projects:
+
+  - name: che-theia
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che-theia.git'
+
+components:
+
+  - id: eclipse/che-theia/next
```

Comment:
I do not agree with using the 'next' tag. Let us remove 'cheEditor' and will be using the default(which depends on infrastructure)

Reply:
If we remove `cheEditor` at all, we will not be able to run commands in Theia editor container.

Reply:
Just to compare, you can take a look at the devfile to develop Che-Theia https://github.com/eclipse/che-theia/blob/master/devfiles/che-theia-all.devfile.yaml